js/tootils/src/index.ts

```diff
@@ -163,29 +163,40 @@ export * from "./render";
 
 export const drag_and_drop_file = async (
 	page: Page,
-	selector: string,
+	selector: string | Locator,
```

Comment:
Tweaked this util:

1. so it can take an instance of `Locator` as an alternative to a string selector
2. so it can take a number of files to upload to the input